tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -88,10 +88,11 @@ object CodeGen {
     var currentBlockName: Local = _
     var currentBlockSplit: Int  = _
 
-    val copies    = mutable.Map.empty[Local, Val]
-    val deps      = mutable.Set.empty[Global]
-    val generated = mutable.Set.empty[String]
-    val builder   = new ShowBuilder
+    val copies           = mutable.Map.empty[Local, Val]
+    val deps             = mutable.Set.empty[Global]
+    val generated        = mutable.Set.empty[String]
+    val externSigMembers = mutable.Map.empty[Sig, Global.Member]
+    val builder          = new ShowBuilder
```

Comment:
Question: Does it make sense to make these mutable things `private`? In case it does, should this be part of this pull request?

Reply:
The class itself is private is private, so it doesn't matter much, but feel free to make them explicitly private if they are not used by other classes in the same file. It won't hurt :)